rosdep/sources.list.d/20-default.list

```diff
@@ -1,4 +1,5 @@
 # os-specific listings first
+yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/oe-pkg.yaml
```

Comment:
Although I think there's some value in having a separate file for OE maintenance. I suspect that the maintenance will be low for these files and as such it would actually be better to integrate this into the base.yaml. It's actually a bit of a pain to have to cross reference into another file when trying to cross reference between platforms. 

This will also collapse the extra lines again as they keys will already exist in base.yaml so it will just be one extra line per key for the oe: line.

Reply:
Agree; good point.